src/connect/index.js

```diff
@@ -0,0 +1,121 @@
+const amqplib = require('amqplib');
+
+const { createChannel } = require('../createChannel');
+const { createLogger } = require('../logger');
+const { prepareContextData } = require('../prepareContextData');
+const { MikoRpc } = require('../mikoRpc');
+
+
+/**
+ * Connects to AMQ and returns MikoRPc object through a promise
```

Comment:
```suggestion
 * Connects to RabbitMQ and returns MikoRPC object through a promise
```
AMQP is the protocol.